README.rst

```diff
@@ -190,6 +190,23 @@ Maintainers of client repos are responsible for applying the changes that happen
 
 2. Don't forget to commit the changes the command above makes.
 
+Run pre-install custom commands
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+
+You may want to add custom steps prior to the setup defined in `./travis.sh <./travis.sh>`_. Example:
+
+* A device driver package X in your repository or in your repository's dependency requires a prorietary library installed. This library is publicly available, but not via apt or any package management system and thus the only way you can install it is in a classic way (unzip, run installer etc.) (`More discussion <<https://github.com/ros-industrial/industrial_ci/issues/14>`_)
+
+In this case, add `source`d scripts before travis.sh gets called (see below for an example). 
+
+::
+
+  script: 
+    - source ./your_custom_PREprocess.sh
+    - source .ci_config/travis.sh
+    - source ./your_custom_POSTprocess.sh
+
+In above case, in both `.ci_config/travis.sh` and `your_custom_POSTprocess.sh` the environment is kept from previous script(s), so whatever done in previous scripts remain. 
```

Comment:
Only language (but not a native speaker myself): "In *the* above case, ..".

and: "whatever *is* done in previous scripts remain*s*".